app/reduxContent/settings/reducers.js

```diff
@@ -82,6 +91,46 @@ export default handleActions(
       }
       return state;
     },
+    [ADD_PATH]: (state, action) => {
+      const newPath = action.path;
+      const pathsList = state.get('pathsList');
+      const indexFound = pathsList.findIndex(item => {
```

Comment:
const indexFound = pathsList.findIndex(item => {
        return item.get('label') === newPath.label;
      });

to
const indexFound = pathsList.findIndex(item => item.get('label') === newPath.label);